### PR TITLE
finished it

### PR DESCRIPTION
**Pull request by dbvu88**
@NickStrick

### CODE REVIEW

File: assignments/arrays.js [hunk 1]
```diff
@@ -63,32 +63,52 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+const car33 = inventory[33];
+console.log(`Car 33 is a ${car33.car_year} ${car33.car_make} ${car33.car_model}` );
```
Comment (NickStrick) [inline assignments/arrays.js:67]: remember array index's start at 0, so to get car 33 you actually need to set it to inventory[33-1]

File: assignments/arrays.js [hunk 2]
```diff
@@ -63,32 +63,52 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
...
 let BMWAndAudi = [];
-console.log();
+
+for (let i = 0; i < inventory.length; i++) {
+    if ( inventory[i].car_make === "BMW" || inventory[i].car_make === "Audi" ) {
+        BMWAndAudi.push(inventory[i]);
+    }
+}
+
+console.log(JSON.stringify(BMWAndAudi));
```
Comment (NickStrick) [inline assignments/arrays.js:111]: the rest of this file looks good!

File: assignments/function-conversion.js [hunk 3]
```diff
@@ -1,30 +1,32 @@
...
+};
+myFunction();
 
-// let anotherFunction = function (param) {
-//   return param;
-// };
-// anotherFunction("Example");
+let anotherFunction = (param) => {
+  return param;
+};
```
Comment (NickStrick) [inline assignments/function-conversion.js:10]: these all work and look good, but remember that a big feature of arrow functions is to get rid of brackets and return statements. this function can look like this:
```
let anotherFunction = (param) => param;

**Review by NickStrick — APPROVED** [review]
looks great. 👍

File: assignments/arrays.js (hunk 1 above)
Comment (dbvu88) [reply assignments/arrays.js:67]: @NickStrick I actually called the index 32 originally, but then I notice the word "zeroth", and I was like "are they trying to trick me" and change it to 33

File: assignments/function-conversion.js (hunk 3 above)
Comment (dbvu88) [reply assignments/function-conversion.js:10]: ahh forgot that, thanks

**Pull request merged**